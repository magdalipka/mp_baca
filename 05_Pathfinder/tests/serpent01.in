10 10
0 1 0 0 0 1 0 0 0 1
0 1 0 1 0 1 0 1 0 1
0 1 0 1 0 1 0 1 0 1
0 1 0 1 0 1 0 1 0 1
0 1 0 1 0 1 0 1 0 1
0 1 0 1 0 1 0 1 0 1
0 1 0 1 0 1 0 1 0 1
0 1 0 1 0 1 0 1 0 1
0 1 0 1 0 1 0 1 0 1
0 0 0 1 0 0 0 1 0 0
2
r 0 9 9 0
i 0 9 9 0

