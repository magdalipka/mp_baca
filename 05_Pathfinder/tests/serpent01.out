r S S S S S S S S S E E N N N N N N N N N E E S S S S S S S S S E E N N N N N N N N N E E S S S S S S S S S E
i S S S S S S S S S E E N N N N N N N N N E E S S S S S S S S S E E N N N N N N N N N E E S S S S S S S S S E
